eve/main.yml

```diff
@@ -24,7 +24,7 @@ stages:
           command: yarn install --frozen-lockfile
       - ShellCommand:
           name: run lint
-          command: yarn run --silent lint -- --max-warnings 0
+          command: echo "linter is disabled temporarily ()" || yarn run --silent lint -- --max-warnings 0
```

Comment:
should this still be disabled?

Reply:
Yeah, according to @rahulreddy we will address the eslint issues on a later stage, we need to focus on the Node 16 Update now as Node 10 is about to reach EOL. Instead of disabling individual lines, I decided to disable it entirely and show a message.

Reply:
gotcha